docs/api-reference/core/layer.md

```diff
@@ -562,6 +571,10 @@ Used to update the layers [`state`](/docs/api-reference/core/layer.md#state) obj
 
 Used to update the settings of shader modules.
 
+##### `throw`
+
+Send an `Error` object to the error handling pipeline.
```

Comment:
I assume the purpose is for this to be used in layer callbacks to ensure that errors flow through our system instead of break the layer management?

Can't we just wrap such functions in a `try/catch` and automatically do this on behalf of the user and avoid new methods?

And a section about error handling in layers.

Reply:
This is used internally inside `catch` blocks. This section about layer methods notes that they are for custom layer implementation only.

Reply:
Got it, starting to make good sense. Maybe articulate the intended usage in the docs (`Used to propagate errors thrown in layers to the deck.gl error handling system. This prevents rendering to be interrupted by exceptions in layers` and maybe show a few lines of sample code for customer layer writers)?